core/src/main/java/com/dtolabs/rundeck/core/execution/workflow/steps/DynamicPropertiesStepPlugin.java

```diff
@@ -0,0 +1,7 @@
+package com.dtolabs.rundeck.core.execution.workflow.steps;
+
+import java.util.Map;
+
+public interface DynamicPropertiesStepPlugin {
```

Comment:
Why are there two interfaces (`DynamicPropertiesStepPlugin` and `DynamicProperties`) that are basically the same?